docker-compose.yml

```diff
@@ -37,13 +37,13 @@ services:
       "
     env_file: .env
     environment:
+      FLASK_APP: app/__init__.py
```

Comment:
Can just be:

```suggestion
      FLASK_APP: app
```

But no big problem either way

Reply:
The dockerfile also feeds into pullpreview, so we'll need to differentiate between these two regardless. I think it's fine to have in this docker compose file